rsciio/quantumdetector/_api.py

```diff
@@ -269,18 +275,26 @@ def load_mib_data(
             count=mib_prop.xy,
             offset=mib_prop.offset,
         )
+
     else:  # pragma: no cover
-        raise ValueError("`path` must be a str or a buffer.")
+        raise TypeError("`path` must be a str or a buffer.")
+
+    headers = data["header"]
+    data = data["data"]
+    if not return_mmap:
+        if lazy:
+            data = da.from_array(data)
```

Comment:
Add in chunks parameter here?

Reply:
Yes, quite often, we read from files using chunking and there is limited advantage in specifying the chunking but in this case, it make sense!

We should document it and make more consistent across file readers.